tests/test_variogram_structured.py

```diff
@@ -1,14 +1,21 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
 This is a unittest of the variogram module.
 """
 from __future__ import division, absolute_import, print_function
 
+import sys
 import unittest
 import numpy as np
 from gstools import variogram
 
+PY3 = sys.version_info[0] == 3
+# in python3 "long" was replaced with "int"
```

Comment:
nice one!

Reply:
It's strange because the cython file itself isn't affected by that! I have to read about that.